cpp/benchmarks/join/join_common.hpp

```diff
@@ -147,6 +145,17 @@ void BM_join(state_type& state, Join JoinFunc)
   cudf::table_view left_table(
     {left_key_column0->view(), left_key_column1->view(), *left_payload_column});
 
+  auto table_bytes = [](cudf::table_view tbl) {
+    size_t bytes = 0;
+    for (cudf::size_type i = 0; i < tbl.num_columns(); i++) {
+      auto col = tbl.column(i);
+      bytes += (sizeof(Key) * col.size()) +
+               (col.nullable() ? std::ceil(static_cast<double>(col.null_count()) / 8) : 0);
+    }
+    return bytes;
+  };
+  auto const data_size = table_bytes(right_table) + table_bytes(left_table);
```

Comment:
```suggestion
  auto const join_input_size = table_bytes(right_table) + table_bytes(left_table);
```
maybe `join_input_size` for better clarity